PROVISIONING-METAL.md

```diff
@@ -152,13 +189,65 @@ addresses = ["192.168.14.5/24"]
 to = "10.10.10.0/24"
 from = "192.168.14.5"
 via = "192.168.14.25"
+
+# A bond is a network device that is of `kind` `bond`
+[bond0]
+kind = "bond"
+# Currently `active-backup` is the only supported option
+mode = "active-backup"
+# In this case, the vlan will have addressing, the bond is simply there for use in the vlan
+dhcp4 = false
+dhcp6 = false
+# The first interface in the array is considered `primary` by default
+interfaces = ["eno11", "eno12"]
+
+[bond0.monitoring]
+miimon-frequency-ms = 100 # 100 milliseconds
+miimon-updelay-ms = 200 # 200 milliseconds
+miimon-downdelay-ms = 200 # 200 milliseconds
+
+[bond1]
+kind = "bond"
+mode = "active-backup"
+interfaces = ["eno51" , "eno52", "eno53"]
+min-links = 2 # Optional min-links 
+dhcp4 = true
+
+[bond1.monitoring]
+arpmon-interval-ms = 200 # 200 milliseconds
+arpmon-validate = "all"
+arpmon-targets = ["192.168.1.1", "10.0.0.2"]
+
+# A vlan is a network device that is of `kind` `vlan`
+# VLAN42 is the name of the device, can be anything that is a valid network interface name
+[VLAN42]
+kind = "vlan"
+device = "bond0"
+id = 42
+dhcp4 = true
+
+[internal_vlan]
+kind = "vlan"
+device = "eno2"
+id = 1234
+dhcp6 = true
 ```
 
-**An additional note on network device names**
+#### **An additional note on network device names**
 
 Interface name policies are [specified in this file](https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/release/80-release.link#L6); with name precedence in the following order: onboard, slot, path.
 Typically on-board devices are named `eno*`, hot-plug devices are named `ens*`, and if neither of those names are able to be generated, the “path” name is given, i.e `enp*s*f*`.
 
+#### Networking configuration versions and Releases
+
+Older networking configuration versions (such as `1` or `2`) are supported in newer releases. In order to use a newer version, the following table provides guidance on what release first enabled the version.
+
+| Network Configuration Version | First Release                                                                   |
+|-------------------------------|---------------------------------------------------------------------------------|
+| Version 1                     | [v1.9.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.9.0)   |
+| Version 2                     | [v1.10.0](https://github.com/bottlerocket-os/bottlerocket/releases/tag/v1.10.0) |
+| Version 3                     | Unreleased                                                                      |
```

Comment:
I originally had it that way. We had expected to merge this a while ago and have other edits in the repo since then and flip then. Since we are still in PR, it makes sense to just change it for the next release.

Reply:
I decided to leave this for now, I'll be tracking quite a bit for the next release and plan to make sure this makes it into the release changes for whatever release it does make.